beryllium/src/main/java/de/b3/bubatz_service/services/control/ServiceControl.java

```diff
@@ -31,4 +33,13 @@ public GetService createService(PostService postService) {
 
         return ServiceMapper.map(saved);
     }
+
+    public void bookService(PatchService service) {
+        final Service entity = this.repository.findById(service.getId())
+                .orElseThrow(() -> new EntityNotFoundException("The Service with Id" + service.getId() + "could not been found"));
+
+        entity.setAvailable(service.getState());
```

Comment:
Not sure wo die setAvailable Methode herkommt (also klar, aus der Service-Klasse) aber weiß nicht wann diese Änderung gemacht wurde, weil auf dem Master gibts die noch net.
Vllt bin ich aber halt auch einfach dumm